.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /root
     docker:
-      - image: aarondl0/sqlboiler-test:latest
+      - image: quay.io/glerchundi/sqlboiler-test:v3
```

Comment:
I've built and published this tag to `aarondl0/sqlboiler-test:v3`, please change to reflect this.

The old one will reside on aarondl0/sqlboiler-test:v2 (and it also pulls as latest since I pushed with a specific tag).

Reply:
Ok.